src/engraving/rw/read410/readcontext.h

```diff
@@ -237,6 +239,7 @@ class ReadContext
     SettingsCompat _settingsCompat;
 
     TimeSigMap m_compatTimeSigMap;
+    std::set<Pid> m_stylesToSkip;
```

Comment:
You can use PropertyIdSet here. It may work faster as it uses std::unordered_set and you only need to check if it contains a key or not